Source/WebCore/page/DebugPageOverlays.cpp

```diff
@@ -296,7 +296,7 @@ static Vector<Path> pathsForRegion(const InteractionRegion& region)
     static constexpr float radius = 4;
 
     Vector<FloatRect> rects;
-    for (auto rect : region.rectsInContentCoordinates)
+    for (auto rect : region.regionInLayerCoordinates.rects())
         rects.append(rect);
     return PathUtilities::pathsWithShrinkWrappedRects(rects, std::max(region.borderRadius, radius));
```

Comment:
Nit - can we use `map()` here, or `.reserveInitialCapacity()` avoid resizing the `Vector<FloatRect>` while appending to it?

Reply:
Also weird! Before this patch, this loop made no sense (because `rectsInContentCoordinates` was Vector<FloatRect>). I guess maybe it had more code inside it at some point. Switching to map, yes!